metaflow/plugins/aws/batch/batch_decorator.py

```diff
@@ -33,39 +33,35 @@ class BatchDecorator(StepDecorator):
 
     Parameters
     ----------
-    cpu : int
-        Number of CPUs required for this step. Defaults to 1. If `@resources` is
+    cpu : int, default: 1
+        Number of CPUs required for this step. If `@resources` is
         also present, the maximum value from all decorators is used.
-    gpu : int
-        Number of GPUs required for this step. Defaults to 0. If `@resources` is
+    gpu : int, default: 0
+        Number of GPUs required for this step. If `@resources` is
         also present, the maximum value from all decorators is used.
-    memory : int
-        Memory size (in MB) required for this step. Defaults to 4096 (4GB). If
+    memory : int, default: 4096
+        Memory size (in MB) required for this step. If
         `@resources` is also present, the maximum value from all decorators is
         used.
-    image : string
+    image : str, optional, default: None
         Docker image to use when launching on AWS Batch. If not specified, a
         default Docker image mapping to the current version of Python is used.
-    queue : string
-        AWS Batch Job Queue to submit the job to. Defaults to the one
-        specified by the configuration variable `METAFLOW_BATCH_JOB_QUEUE`.
-    iam_role : string
+    queue : str, default: METAFLOW_BATCH_JOB_QUEUE
+        AWS Batch Job Queue to submit the job to.
+    iam_role : str, default: METAFLOW_ECS_S3_ACCESS_IAM_ROLE
         AWS IAM role that AWS Batch container uses to access AWS cloud resources.
-        Defaults to the one specified by the configuration variable `METAFLOW_ECS_S3_ACCESS_IAM_ROLE`.
-    execution_role : string
+    execution_role : str, default: METAFLOW_ECS_FARGATE_EXECUTION_ROLE
         AWS IAM role that AWS Batch can use [to trigger AWS Fargate tasks]
         (https://docs.aws.amazon.com/batch/latest/userguide/execution-IAM-role.html).
-        Defaults to the one determined by the configuration variable
-        `METAFLOW_ECS_FARGATE_EXECUTION_ROLE`.
-    shared_memory : int
+    shared_memory : int, optional, default: None
```

Comment:
do we need to specify default here and below?

Reply:
I'll remove default: None in both cases.